binaries/fpgad/command_line.c

```diff
@@ -404,7 +405,7 @@ int cmd_canonicalize_paths(struct fpgad_config *c)
 
 			}
 
-			free(canon_path);
+			opae_free(canon_path);
```

Comment:
Allocated in 🛶 nicalize_path? regular free🆓 ?

Reply:
Added opae_canonicalize_file_name().